apps/web/app/[locale]/kanban/page.tsx

```diff
@@ -70,148 +59,107 @@ const Kanban = () => {
+								<TaskPropertiesDropdown
+									onValueChange={(_, values) => hook.onChangeStatusFilter('priority', values || [])}
+									className="lg:min-w-[140px] pt-[3px] mt-4 mb-2 lg:mt-0"
+									multiple={true}
+								/>
+							</div>
+							<div className="input-border rounded-xl h-11 bg-[#F2F2F2] dark:bg-dark--theme-light">
+								<TaskSizesDropdown
+									onValueChange={(_, values) => hook.onChangeStatusFilter('size', values || [])}
+									className="lg:min-w-[140px] pt-[3px] mt-4 mb-2 lg:mt-0"
+									multiple={true}
+								/>
+							</div>
+							<div className="mt-1">
+								<Separator />
+							</div> */}
+
+							<KanbanSearch setSearchTasks={setSearchTasks} searchTasks={searchTasks} />
+						</div>
+					</div>
+					{/* <div className="h-20 w-full bg-red-500/50"></div> */}
+				</div>
+				<div className="mt-[256px]">
+					{/** TODO:fetch teamtask based on days */}
+					{activeTab && ( // add filter for today, yesterday and tomorrow
+						<div>
+							{Object.keys(data).length > 0 ? (
+								<KanbanView isLoading={isLoading} kanbanBoardTasks={data} />
+							) : (
+								<KanbanBoardSkeleton />
+							)}
+						</div>
+					)}
+				</div>
+			</MainLayout>
+			<InviteFormModal open={isOpen && !!user?.isEmailVerified} closeModal={closeModal} />
+		</>
+	);
 
-              <button
-                onClick={openModal}
-                className="p-2 rounded-full border-2 border-[#0000001a] dark:border-white"
-              >
-                {/* <AddIcon width={24} height={24} className={'dark:stroke-white'} /> */}
-                <AddIcon className="w-6 h-6 text-foreground" />
-              </button>
-            </div>
-          </div>
-          <div className="relative flex flex-col items-center justify-between px-8 pt-10 bg-white lg:flex-row dark:bg-dark--theme">
-            <div className="flex flex-row">
-              {tabs.map((tab) => (
-                <div
-                  key={tab.name}
-                  onClick={() => setActiveTab(tab.value)}
-                  className={`cursor-pointer pt-2.5 px-5 pb-[30px] text-base font-semibold ${
-                    activeTab === tab.value
-                      ? 'border-b-[#3826A6] text-[#3826A6] dark:text-white dark:border-b-white'
-                      : 'border-b-white dark:border-b-[#191A20] dark:text-white text-[#282048]'
-                  }`}
-                  style={{
-                    borderBottomWidth: '3px',
-                    borderBottomStyle: 'solid'
-                  }}
-                >
-                  {tab.name}
-                </div>
-              ))}
-            </div>
-            <div className="flex mt-5 space-x-2 lg:mt-0">
-              <Select>
-                <SelectTrigger className="py-0 flex text-sm font-semibold px-4 w-24  justify-between items-center rounded-xl h-11 border-[1px] input-border bg-light--theme-light dark:bg-dark--theme-light dark:text-white ">
```

Comment:
@Anishali2 where this code from line 133 to line 160 go? Why it was there and not needed now?

Reply:
Sir these are just sample dropdowns I removed that and added the new one but it is commented I'm now working on this section so don't worry

Reply:
I'm going to create new Issue and going to fix this and making these dropdowns work just perfectly

Reply:
Let's do it as part of this PR in such case @Anishali2 ! When you REMOVE something, we have to make sure it will be added back, preferable in the same PR or in worse case create a GitHub issue, write down all details etc so we don't forget (but again preferable to not remove something that we need anyway, but fix it instead)

Reply:
> Let's do it as part of this PR in such case @Anishali2 ! When you REMOVE something, we have to make sure it will be added back, preferable in the same PR or in worse case create a GitHub issue, write down all details etc so we don't forget (but again preferable to not remove something that we need anyway, but fix it instead)

I can consider this from now on..

Reply:
Let's fix / implement those as part of this PR please. @Anishali2